### ISSUE TITLE
Domains hosting script not reachable (DE)

### ISSUE THREAD

**Issue by blacktar** [piranha/infowarship#10]
https://infowarship.com/script.js and https://infowarship.pages.dev/go are not working.

**Comment by sansolovyov** [comment]
yeah we've seen that stuff is absolutely ineffective, so we shut them down at some point
